libs/api/domains/driving-license-book/src/models/drivingLicenseBookSchoolType.response.ts

```diff
@@ -0,0 +1,15 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+@ObjectType()
+export class DrivingSchoolType {
+  @Field()
```

Comment:
This is fine ... since this object will most likely not be mutated and thus the Apollo cache doesn't need to know how to construct and merge the incoming changes ... but always good to specify IDs in GraphQL.

Reply:
The schoolTypeId will always be 1, 2 or 3. Should that kind of response be marked with ID?

Reply:
Yes ... if I'm understanding you correctly, then there will only be at most 3 types of this entity in the Apollo cache.